spec/lib/secure_headers/headers/expect_certificate_transparency_spec.rb

```diff
@@ -3,13 +3,13 @@
 
```

Comment:
I also updated the name of this test, as "expect certificate" was ever so slightly confusing :smile:.

Reply:
Thanks! I think this was my bad from when I had it named `expect_ct_spec` => `expect_certificate_spec` 😛